lamp/charts/mariadb/templates/_helpers.tpl

```diff
@@ -0,0 +1,15 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{define "mariadb.name"}}{{default "mariadb" .Values.nameOverride | trunc 24 }}{{end}}
```

Comment:
No, doesn't look like it! I guess the name override is for upgrade purposes or something?